tests/playwright/src/model/pages/create-cluster-page.ts

```diff
@@ -0,0 +1,63 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import { expect as playExpect, type Locator, type Page } from '@playwright/test';
+
+import { BasePage } from './base-page';
+
+export class CreateKindClusterPage extends BasePage {
+  readonly clusterPropertiesInformation: Locator;
+  readonly clusterNameField: Locator;
+  readonly clusterProviderType: Locator;
+  readonly clusterHttpsPort: Locator;
+  readonly clusterHttpPort: Locator;
+  readonly clusterIngressControllerCheckbox: Locator;
+  readonly clusterCreationButton: Locator;
+  readonly goBackButton: Locator;
+  readonly logsButton: Locator;
+
+  constructor(page: Page) {
+    super(page);
+    this.clusterPropertiesInformation = this.page.getByRole('form', { name: 'Properties Information' });
+    this.clusterNameField = this.clusterPropertiesInformation.getByLabel('Name');
+    this.clusterProviderType = this.clusterPropertiesInformation.getByLabel('Provider Type', { exact: true });
+    this.clusterHttpPort = this.clusterPropertiesInformation.getByLabel('HTTP Port', { exact: true });
+    this.clusterHttpsPort = this.clusterPropertiesInformation.getByLabel('HTTPS Port', { exact: true });
+    this.clusterIngressControllerCheckbox = this.clusterPropertiesInformation.getByRole('checkbox');
+    this.clusterCreationButton = this.clusterPropertiesInformation.getByRole('button', { name: 'Create', exact: true });
+    this.goBackButton = this.page.getByRole('button', { name: 'Go back to resources' });
+    this.logsButton = this.page.getByRole('button', { name: 'Show Logs' });
+  }
+
+  public async checkClusterProperties(): Promise<void> {
+    await playExpect(this.clusterPropertiesInformation).toBeVisible();
```

Comment:
@amisskii  I am not sure if we need this kind of check for every element. We gonna check for some once we ie. expect the page to appear after create New button was pressed, then once we gonna start filling some info into fields, etc. whyt @cbr7 ?